specifications/api/swagger.yaml

```diff
@@ -2693,6 +2693,33 @@ x-errInvalidCoinSelection: &errInvalidCoinSelection
       type: string
       enum: ['invalid_coin_selection']
 
+x-errTokenBundleSizeExceedsLimit: &errTokenBundleSizeExceedsLimit
+  <<: *responsesErr
+  title: token_bundle_size_exceeds_limit
+  properties:
+    message:
+      type: string
+      description: |
+        Returned when a user-specified transaction output contains a token
+        bundle whose serialized length is longer than the upper size limit
+        supported by the protocol.
+    code:
+      type: string
+      enum: ['token_bundle_size_exceeds_limit']
+
+x-errTokenQuantityExceedsMaxBound: &errTokenQuantityExceedsMaxBound
+  <<: *responsesErr
+  title: token_quantity_exceeds_max_bound
+  properties:
+    message:
+      type: string
+      description: |
+        Returned when a user-specified transaction output contains a token
+        quantity that exceeds the maximum bound supported by the protocol.
+    code:
+      type: string
+      enum: ['token_quantity_exceeds_max_bound']
```

Comment:
How about calling the errors `too_many_tokens[_in_output]` and `too_large_token`?

Reply:
@Anviking thanks for this comment. I was also a bit unhappy with the naming.

I agree it's very hard to come up with names that are completely unambiguous.

However, I do think that:

- since these errors pertain to **_outputs_**, the name `Output` should probably appear in the name of each error.
- since both errors pertain to **_exceeding some limit_**, it's better to use consistent terminology. I've gone for "exceeds limit" in both cases.

I've revised the names to the following:

| Name | Meaning |
| -- | -- |
| `output_token_quantity_exceeds_limit` | A user-specified transaction **_output_** contains, for some asset, a **_token quantity_** that **_exceeds_** the **_limit_** supported by the protocol. |
| `output_token_bundle_size_exceeds_limit` | A user-specified transaction **_output_** contains a **_token bundle_** whose **_size_** (serialized length) **_exceeds_** the **_limit_** supported by the protocol. |

See: 5abaabda12345e93545eb4fc4fdb6c1efbb217d0